cmd/kube-controller-manager/app/plugins.go

```diff
@@ -72,6 +73,19 @@ func ProbeRecyclableVolumePlugins(flags options.VolumeConfigFlags) []volume.Volu
 		RecyclerTimeoutIncrement: flags.PersistentVolumeRecyclerIncrementTimeoutNFS,
 		RecyclerPodTemplate:      volume.NewPersistentVolumeRecyclerPodTemplate(),
 	}
+
+	rbdConfig := volume.VolumeConfig{
+		RecyclerMinimumTimeout:   flags.PersistentVolumeRecyclerMinimumTimeoutNFS,
```

Comment:
The current method of adding CLI flags for every recycler doesn't scale very well. I was hesitant to add new flags (creating API) that we may deprecate after 1.2